internal/db/schema/migrations/postgres/7/02_job.up.sql

```diff
@@ -0,0 +1,124 @@
+begin;
+
+    create table job_plugin (
+        public_id wt_public_id primary key
+    );
+
+    insert into job_plugin (public_id)
+    values
+        ('jpi_internal');
+
+    comment on table job_plugin is
+        'job_plugin is a table where each row represents a unique plugin that can register jobs.';
+
+    create table job (
+         plugin_id wt_public_id not null
+             constraint job_plugin_fk
+                 references job_plugin(public_id)
+                 on delete cascade
+                 on update cascade,
+         name wt_name not null,
+         description wt_description not null,
+         next_scheduled_run timestamp with time zone not null,
```

Comment:
I think this should be `wt_timestamp`

Reply:
@louisruch , correct me if I'm wrong, but I think this is not a `wt_timestamp` because `wt_timestamp` uses `current_timestamp` as a [default value](https://github.com/hashicorp/boundary/blob/3c994f66f877224fd1d75e6ce3ff4efb3aa9a0ad/internal/db/schema/migrations/postgres/0/01_domain_types.up.sql#L43) which is not the behavior wanted here.

Reply:
Oh sorry I never saw this, yeah @mgaffney is correct we wanted to avoid the default behavior.

Reply:
Ah OK, sounds good. I saw wt_timestamp was with time zone not null and forgot about the default.